css/start-use-css/properties-in-css/task_2/task_3/horiz._and_vertic._menu.css

```diff
@@ -0,0 +1,20 @@
+#menu ul {
+    list-style-type: none;
+    width: 200px;
+}
+
+#menu li {
+    margin-bottom: 5px;
+}
+
+.a {
```

Comment:
Not use short class name, it's important

Reply:
corrected